debian/rules

```diff
@@ -3,3 +3,6 @@
 
 %:
 	dh $@
+
+execute_after_dh_fixperms:
+	chmod 0755 update-motd.d/99-cloud-init-warning
```

Comment:
It shouldn't be required. install should preserve exec bits. But make sure the bits are correct in the source tree.

Reply:
When git-clonning on two different machines (different Ubuntu releases) the script came with the file mode 0755, so we should be fine.